### PR TITLE
[v-hacd] Fix cmake patch for include files

### PR DESCRIPTION
**Pull request by FabienPean**
This PR is an attempt at fixing the v-hacd port. It fixes the target include directory so that the public include file `VHACD.h` can be found in the include path. It also stops installing the private header files in order to reduce clutter.

I think the patch+portfile from the port (and also the CMakeLists from the project repository) could benefit from some rework (cleanup, simplification, fix of shared library), unfortunately I am unable to do it at the moment. The changes are the minimum I had to do to make it work on my machine.

- Which triplets are supported/not supported? Have you updated the CI baseline?
Tested with x64-windows and VS2019. What is the CI baseline ?

Let me know if anything is missing or feel free to take over from here.

### CODE REVIEW

File: ports/v-hacd/fix-cmake.patch [hunk 1]
```diff
@@ -65,13 +65,24 @@ index 46fc1b1..2260fdc 100644
  
 -
  target_include_directories(vhacd PUBLIC
-+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/public/>)
++        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/public/>
++        $<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include/>)
```
Comment (ras0219-msft) [inline ports/v-hacd/fix-cmake.patch:69]:
```suggestion
+        $<INSTALL_INTERFACE:include>)
```
The `INSTALL_INTERFACE` is automatically relative to `CMAKE_INSTALL_PREFIX`

File: ports/v-hacd/fix-cmake.patch [hunk 2]
```diff
@@ -65,13 +65,24 @@ index 46fc1b1..2260fdc 100644
...
  target_include_directories(vhacd PUBLIC
-+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/public/>)
++        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/public/>
++        $<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include/>)
 +
 +target_include_directories(vhacd PRIVATE
    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc/>
-   $<INSTALL_INTERFACE:include/>  # <prefix>/include/mylib
+-  $<INSTALL_INTERFACE:include/>  # <prefix>/include/mylib
++  $<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include/>  # <prefix>/include/mylib
```
Comment (ras0219-msft) [inline ports/v-hacd/fix-cmake.patch:74]: This change shouldn't be needed; INSTALL_INTERFACE in PRIVATE does nothing.

**Review by ras0219-msft — CHANGES_REQUESTED** [review]
Thanks for the PR! Please also modify the `CONTROL` file to introduce a `Port-Version: 1` below the current `Version:` field.

**Comment by FabienPean** [comment]
Thanks for the review, I amended the commit with the requested changes.

**Review by PhoebeHui — APPROVED** [review]
LGTM, thanks for the PR!

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**